maven-core/src/main/java/org/apache/maven/project/collector/MultiModuleCollectionStrategy.java

```diff
@@ -91,7 +105,8 @@ public List<MavenProject> collectProjects( MavenExecutionRequest request ) throw
 
             if ( fallThrough )
             {
-                LOGGER.debug( "Multi module project collection failed:{}"
+                LOGGER.debug(
```

Comment:
It seems to me that these lines are unchanged apart from some whitespace. If that is the case, could you please revert the lines that aren't changed?